packages/@aws-cdk/aws-codepipeline/lib/action.ts

```diff
@@ -38,12 +40,24 @@ export class GlobalVariables {
   public static readonly executionId = '#{codepipeline.PipelineExecutionId}';
 }
 
+/**
+ * How an action is configured
```

Comment:
I think you can also remove some of the linter exceptions, thanks to the comments you added: https://github.com/aws/aws-cdk/blob/e24048ffb1a04bc458d4c220bc4b632379d50c91/packages/%40aws-cdk/aws-codepipeline/package.json#L142-L157